### ISSUE TITLE
Reingest by query tool

### ISSUE THREAD

**Issue by i-like-robots** [Financial-Times/n-es-tools#15]
It would be very useful to have a tool that could reingest a batch of content based on the results of a query to replace [ad-hoc scripts like this](https://github.com/Financial-Times/next-es-interface/wiki/I-need-to-reindex-loads-of-things).

It could accept a query string query and work like this:

```sh
$ n-es-tools reindex eu _missing_:type
```

This would generate the following request URL:

```
http://next-elastic.ft.com/content/item/_search?_source=[]&size=100&q=_missing_:type
```

The matching document IDs can then be plucked and sent for reingestion.

**Comment by i-like-robots** [comment]
I think #18 is a good-enough solution for this problem.

**Comment by i-like-robots** [comment]
Closed by #18 

```sh
```
